src/App.js

```diff
@@ -138,7 +138,7 @@ class App extends React.Component {
         </section>
 
         <section className="sectionBox">
-          <InformationBox 
+          <InformationBox
```

Comment:
tady se mi jen odmazala mezera, nema to co delat s tou funkcionalitou :D ale nechtelo se mi menit lokalni nastaveni formatovani. ta mezera je tam navic a nepatri tam :-) i kdyz to s ni funguje